config/samples/dataplane_v1beta1_openstackdataplane_baremetal_with_ipam.yaml

```diff
@@ -26,10 +26,20 @@ spec:
           value: "True"
         - name: ANSIBLE_VERBOSITY
           value: "2"
+      preProvisioned: false
```

Comment:
This is the default.

Reply:
I know, but this was more about kustomize behaviour if we ever unified to a single gen-edpm-kustomize.sh script. I think it fails with the `replace` op when the value is missing:

https://github.com/openstack-k8s-operators/install_yamls/blob/main/scripts/gen-edpm-kustomize.sh#L70-L72

Reply:
I might restore this, since that kustomize operation isn't actually required